.gitignore

```diff
@@ -1,5 +1,5 @@
 # Python Virtual Environment
-/venv
+/env
```

Comment:
CAVEAT: there might be limits on using this (PyCharm) IDE in certain environments.